### DISCUSSION TITLE
The default value of max-connections configuration key

### DISCUSSION THREAD

**Discussion by tarurar** [rabbitmq/rabbitmq-server#9369] [Other] [answered]
Hi there,
Couldn’t find the information on the default value of `max-connections` configuration key.

Server version is 3.9.13.

**Comment by mkuratczyk** [comment, marked as answer]
By default, there is no limit.

**Comment by lukebakken** [comment]
This isn't very obvious, but this is where defaults are set:

https://github.com/rabbitmq/rabbitmq-server/blob/v3.9.13/deps/rabbit/Makefile#L29

PS version 3.9.13 is completely out of support - https://www.rabbitmq.com/versions.html
